central/sensor/service/pipeline/imageintegrations/pipeline_test.go

```diff
@@ -142,7 +142,8 @@ func Test_matchesECRAuth(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			assert.Equalf(t, tt.want, matchesECRAuth(tt.args.this, tt.args.other), "matchesECRAuth(%v, %v)", tt.args.this, tt.args.other)
+			assert.Equalf(t, tt.want, matchesECRAuth(tt.args.this, tt.args.other),
+				"matchesECRAuth(%q, %q)", tt.args.this.String(), tt.args.other.String())
```

Comment:
I suppose this one came on the radar screen because of the arguments to `machesECRAuth`, which are generated protos.
Would it make sense to extract the function call and apply the assertion on the function result ?

Reply:
Fixed